tests/python/quantization/test_quantization.py

```diff
@@ -409,11 +441,19 @@ def test_quantize_sym_with_calib():
     sym = get_fp32_sym()
     offline_params = [name for name in sym.list_arguments()
                       if not name.startswith('data') and not name.endswith('label')]
-    qsym = mx.contrib.quant._quantize_symbol(sym, offline_params=offline_params)
-    requantize_op_names = ['requantize_conv', 'requantize_fc']
-    th_dict = {'conv_output': (np.random.uniform(low=100.0, high=200.0), np.random.uniform(low=100.0, high=200.0)),
-               'fc_output': (np.random.uniform(low=100.0, high=200.0), np.random.uniform(low=100.0, high=200.0))}
-    op_name_to_th_name = {'requantize_conv': 'conv_output', 'requantize_fc': 'fc_output'}
+    if is_test_for_mkldnn():
+        dtype = 'uint8'
+        requantize_op_names = ['requantize_conv']
+        th_dict = {'conv_output': (np.random.uniform(low=100.0, high=200.0), np.random.uniform(low=100.0, high=200.0))}
+        op_name_to_th_name = {'requantize_conv': 'conv_output'}
+    else:
+        dtype = 'int8'
+        requantize_op_names = ['requantize_conv', 'requantize_fc']
+        th_dict = {'conv_output': (np.random.uniform(low=100.0, high=200.0), np.random.uniform(low=100.0, high=200.0)),
+                   'fc_output': (np.random.uniform(low=100.0, high=200.0), np.random.uniform(low=100.0, high=200.0))}
+        op_name_to_th_name = {'requantize_conv': 'conv_output', 'requantize_fc': 'fc_output'}
```

Comment:
The expected output should not depend on the backend. Also, the fully connected output is missing above

Reply:
Currently mkldnn doesn't support int8 for fc layer so we made separate logic for mkldnn to reflect real MKLDNN use case, once it is supported we can use the same logic.